posts/views.py

```diff
@@ -86,6 +91,39 @@ def PostRepostView(request, pk=None):
     return redirect('posts:post', pk=repost.pk)
 
 
+def ParseContent(content):
```

Comment:
Damn @MetalMatze, why did you even start this? :D Now everyone names their functions in UpperCamelCase! We should keep it this way for now and mass-rename them all later to `snake_case`.